packages/test-utils/src/wrapper.js

```diff
@@ -141,6 +146,13 @@ export default class Wrapper implements BaseWrapper {
       this.vm.$destroy()
       throwIfInstancesThrew(this.vm)
     }
+
+    // allow garbage collection
+    if (Wrapper._instances) {
+      Wrapper._instances = Wrapper._instances.filter(
```

Comment:
in combination with the hook callback this has `O(n²)` complexity (`n` being the wrapper instances). maybe we want to get rid of this.